.eleventy.js

```diff
@@ -124,10 +132,10 @@ module.exports = function (eleventyConfig) {
     }
     return `
       <ul class="doc-links">
-        ${stageChip} ${figmaLink}
+        ${stageChip}
         <li class="github-link">
           <a href="${githubLink}" target="_blank" rel="nofollow" aria-label="${langStrings[locale].github} ${langStrings[locale].newtab}">${langStrings[locale].github}</a>
-        </li>
+        </li> ${figmaLink}
```

Comment:
Sorry very picky formatting comment again ( :D ) but can we move this to a new line to make clear that it's not related to the Github link?

Reply:
I know it's ugly but the reason the figma link is on that line is to prevent the markdown renderer from placing a `<p>` tag in between the `<li>` elements

Reply:
Gotcha, disregard the comment then :)